packages/frontend/core/src/modules/peek-view/view/doc-preview/doc-peek-view.tsx

```diff
@@ -143,6 +156,20 @@ function DocPeekPreviewEditor({
     peekView.close();
   }, [doc, peekView, workbench]);
 
+  const value = useMemo(() => {
+    const fontStyle = fontStyleOptions.find(
+      option => option.key === settings.fontFamily
+    );
+    if (!fontStyle) {
+      return cssVar('fontSansFamily');
+    }
+    const customFontFamily = settings.customFontFamily;
+
+    return customFontFamily && fontStyle.key === 'Custom'
+      ? `${customFontFamily}, ${fontStyle.value}`
+      : fontStyle.value;
+  }, [settings.customFontFamily, settings.fontFamily]);
+
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +93 to +172]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->How about extract these logic to a wrapper like `<CustomEditorSettingProvider>`   and wrap editor where needed:

- center peek
- detail page
- maybe share page?

The wrapper only handle custom editor settings, and only attach css variables, may implemented with radix `<Slot>` or css `display: contents`<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/9274?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
I made some changes based on your suggestions. Does this meet your requirements?